Makefile

```
@@ -0,0 +1,78 @@
+BINARY_NAME=nav-tracker
+BUILD_DIR=build
+PORT=8080
+
+.PHONY: help build clean test run docker-build docker-run lint
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)